src/components/ui/MatchListItem.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import Avatar from "./Avatar";
+import "styles/views/Matches.scss";
+
+export default class MatchListItems extends Component {
+    constructor(props) {
```

Comment:
Is it possible to use webHooks (useState) inside of classes?

Reply:
i do not know, but think so. I think the main reason of the MatchListItems is to too route to the chat or the userprofile, I dont know if we need webHooks there?